samples/TodoList/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Sun Apr 09 20:23:19 PDT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
```

Comment:
Why not current, e.g. 4.10.2 or 5?

Reply:
Same here. I don't see any problem with upgrading to the latest version of Gradle.
Though I don't see any benefits of it. Whatever version we choose users will end up with different in a month or two.

Reply:
@dryganets having consistent versions across samples is a solid reason. I was mostly just curious. 4.10.2 has the disk space benefit of cache expiry but otherwise I also agree it's more or less a treadmill. It does seem the react native style these days is have a property and then in build.gradle use the property if it exists or use a defined version if not, so developers can control their own dependencies if desired